.github/build/main.py

```diff
@@ -0,0 +1,77 @@
+import yaml
+import os
+import requests
+
+def main():
+    with open("./config/opendatahub-io/org.yaml", "r") as f:
+        data = yaml.load(f, Loader=yaml.FullLoader)
+        new_odh_org = data.get("orgs").get("opendatahub-io")
+    os.system("git fetch --all")
+    os.system("git checkout main")
+
+    with open("./config/opendatahub-io/org.yaml", "r") as f:
+        data = yaml.load(f, Loader=yaml.FullLoader)
+        old_odh_org = data.get("orgs").get("opendatahub-io")
+
+    pr_number = os.environ.get("PR_NUMBER")
+    affected_teams = get_affected_teams(old_odh_org, new_odh_org)
```

Comment:
What happens if no team is being affected and the PR is just to add someone to the org? I _think_ that case is currently not coverer by this script.